protool/__init__.py

```diff
@@ -65,6 +85,19 @@ def profile_type(self) -> ProvisioningType:
 
         raise Exception("Unable to determine provisioning profile type")
 
+    @property
+    def developer_certificates(self) -> [Certificate]:
+        """Returns developer certificates as a list of PyOpenSSL X509""" 
+        dev_certs = []
+        for item in self._contents.get("DeveloperCertificates"):
+            try:
+                certificate = Certificate(item)
+                dev_certs.append(certificate)
+            except Exception as cert_exception:
+                print(f"Could not load certificate due to an error: {cert_exception}")
```

Comment:
Just stick with exceptions rather than printing out (that way this can be effectively used as a module).